tests/unit/s2n_tls13_support_test.c

```diff
@@ -28,37 +28,36 @@
 int main(int argc, char **argv)
 {
     BEGIN_TEST();
-    EXPECT_SUCCESS(s2n_disable_tls13_in_test());
-
-    /* TLS 1.3 is not used by default */
-    EXPECT_FALSE(s2n_use_default_tls13_config());
+    EXPECT_SUCCESS(s2n_enable_tls13_in_test());
+    /* TLS 1.3 is used by default */
+    EXPECT_TRUE(s2n_use_default_tls13_config());
 
-    /* TLS1.3 is not supported or configured by default */
+    /* TLS1.3 is supported by default */
```

Comment:
This is not a true statement, which is a bad sign for a test. s2n-tls still does NOT support TLS1.3 by default, unless s2n-tls is in FIPS mode. You've forced the default support via s2n_enable_tls13_in_test.